code/modules/atmospherics/machinery/components/binary_devices/valve.dm

```diff
@@ -5,7 +5,7 @@ It's like a regular ol' straight pipe, but you can turn it on and off.
 /obj/machinery/atmospherics/components/binary/valve
 	icon_state = "mvalve_map"
 	name = "manual valve"
-	desc = "A pipe valve"
+	desc = "A pipe valve."
```

Comment:
be a bit more descriptive here please. Tell us it's function

Reply:
I was just adding a period though why do I have to revamp its desc

Reply:
leave things better than when you found them :^)